Tests/SafeDIToolTests/SafeDIToolTests.swift

```diff
@@ -1716,6 +1716,114 @@ final class SafeDIToolTests: XCTestCase {
         )
     }
 
+    func test_run_writesConvenienceExtensionOnRootOfTree_whenRootInstantiatesAPropertyForwardedByAChild_doesNotRequirePuttingInstantiatedPropertyBeforeForwardingInstantiator() async throws {
```

Comment:
prior to the above changes, this test output compiling code, but this code was ordered sub-optimally.